<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emcDialog</class>
 <widget class="QDialog" name="emcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ear Masa Cob</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Parameters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="libraryGroup">
         <property name="title">
          <string>Spectral Library</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QgsFileWidget" name="browseLibrary" native="true">
            <property name="toolTip">
             <string>Browse for the spectral library</string>
            </property>
            <property name="dialogTitle" stdset="0">
             <string>Select a spectral library</string>
            </property>
            <property name="filter" stdset="0">
             <string>ENVI Spectral Library (*.sli);;All Files (*)</string>
            </property>
            <property name="defaultRoot" stdset="0">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="scaleLayout">
            <item>
             <widget class="QLabel" name="label1">
              <property name="toolTip">
               <string>The reflectance scale factor is automatically detected. It is the number by which the array must be divided to get reflectance values between 0 and 1. The user can change this value here.</string>
              </property>
              <property name="text">
               <string>Reflectance scale factor:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="libraryScaleValue">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>The reflectance scale factor is automatically detected. It is the number by which the array must be divided to get reflectance values between 0 and 1. The user can change this value here.</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="classLayout">
            <item>
             <widget class="QLabel" name="classLabel">
              <property name="text">
               <string>Metadata element with endmember classes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="classDropDown">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Choose the metadata element that contains the classification used in MESMA</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="constraintsGroup">
         <property name="title">
          <string>Set Constraints to calculate RMSE (advanced)</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QGroupBox" name="minFracCheck">
            <property name="styleSheet">
             <string notr="true">QGroupBox { border: none}</string>
            </property>
            <property name="title">
             <string>Min fraction:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>25</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QDoubleSpinBox" name="minFracValue">
               <property name="readOnly">
                <bool>false</bool>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::UpDownArrows</enum>
               </property>
               <property name="minimum">
                <double>-0.500000000000000</double>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>-0.050000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="maxFracCheck">
            <property name="styleSheet">
             <string notr="true">QGroupBox { border: none}</string>
            </property>
            <property name="title">
             <string>Max fraction:</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="leftMargin">
              <number>25</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QDoubleSpinBox" name="maxFracValue">
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="maximum">
                <double>1.500000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>1.050000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QGroupBox" name="maxRMSECheck">
            <property name="styleSheet">
             <string notr="true">QGroupBox { border: none  }</string>
            </property>
            <property name="title">
             <string>Max RMSE:</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <property name="leftMargin">
              <number>25</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QDoubleSpinBox" name="maxRMSEValue">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>0.100000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.001000000000000</double>
               </property>
               <property name="value">
                <double>0.025000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QGroupBox" name="resetCheck">
            <property name="styleSheet">
             <string notr="true">QGroupBox { border: none}</string>
            </property>
            <property name="title">
             <string>Reset fractions to threshold</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="leftMargin">
              <number>25</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Fractions will be reset to the threshold values when models do not meet the fraction constraints. RMSE is then calculated with these corrected values.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="outputGroup">
         <property name="title">
          <string>Output Library</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QgsFileWidget" name="browseOut" native="true">
            <property name="toolTip">
             <string>Choose a name and location for the output library (.sli).</string>
            </property>
            <property name="dialogTitle" stdset="0">
             <string>Select a spectral library (output)</string>
            </property>
            <property name="filter" stdset="0">
             <string>ENVI Spectral Library (*.sli);;All Files (*)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_log">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="logBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="OKClose">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="arrayAction">
   <property name="text">
    <string>...</string>
   </property>
   <property name="iconText">
    <string>...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
